src/docs/markdown/caddyfile/directives/basicauth.md

```diff
@@ -12,6 +12,8 @@ When a user requests a resource that is protected, the browser will prompt the u
 
 Caddy configuration does not accept plaintext passwords; you MUST hash them before putting them into the configuration. The [`caddy hash-password`](/docs/command-line#caddy-hash-password) command can help with this.
 
+After a successful authentication, the `{http.auth.user.id}` placeholder will be available, which contains the authenticated username.
```

Comment:
```suggestion
As with any other authentication backend, after a successful authentication, the `{http.auth.user.id}` placeholder will be available, which contains the authenticated username.
```

Reply:
I think if we word it like that, that'll make users think "what are the other authentication backends?" and they won't find anything useful in the docs. So we should consider how to answer that. We should probably add a section somewhere we can link to.